packages/presentation/presentation_containers/interfaces/publishes_data_view.ts

```diff
@@ -0,0 +1,20 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
+ */
+
+import { DataViewSpec } from '@kbn/data-views-plugin/common';
+import { PublishingSubject } from '@kbn/presentation-publishing';
+
+export interface PublishesDataView {
```

Comment:
This API has a lot of overlap with https://github.com/elastic/kibana/blob/main/packages/presentation/presentation_publishing/interfaces/publishes_data_views.ts. Is there a reason why PublishesDataViews could not be used? If the interface is needed, how about moving into the visualize plugin to avoid overlapping interfaces that do the same thing.

Reply:
Realizing this is leftover from when I was trying to refactor the Visualize editor. This isn't needed when we're just embedding it into the dashboard.

Visualize provided the index pattern through the confusingly named `dataView$` observable, which is not the same as `dataViews`. But that functionality isn't actually needed, so I can remove this.